src/Genre.js

```diff
@@ -0,0 +1,36 @@
+import React, {Component} from "react"
+import { NavLink, Routes, Route, Link } from "react-router-dom";
+import Button from "./Button";
+import GenreDisplay from "./GenreDisplay";
+
+class Genre extends Component {
+    constructor() {
+        super();
+    }
+
+    componentDidMount = () => {
+        this.props.getDetailData();
```

Comment:
I'm wondering if we could actually make this a function, and just bring in getDetailData as a prop and invoke it in our function before the return? 
Probably not worth messing with at this point, since it works, but we may not need this to be a class since we're not even using the constructor....